README.md

```diff
@@ -2,39 +2,296 @@
-| `/field/application/:client-id` [NOT READY YET]                           | `GET`    | Retrieves the field values of all API subscriptions related to a specific application          | `curl -v -X GET    "http://localhost:9650/field/application/hBnFo14C0y4SckYUbcoL2PbFA40a"                              -H "Cache-Control: no-cache"`                                                                                                                                         | 404 NOT_FOUND / 200 OK `{"subscriptions":[{"clientId":"hBnFo14C0y4SckYUbcoL2PbFA40a","apiContext":"ciao-api","apiVersion":"1.0","fieldsId":"f121ffa3-df94-43a0-8235-ac4530f9700a","fields":{"callback-url":"http://localhost:8080/callback","token":"abc59609za2q"}},{"clientId":"hBnFo14C0y4SckYUbcoL2PbFA40a","apiContext":"hello-api","apiVersion":"2.0","fieldsId":"ab1a03cf-8666-4e33-9ef7-07c172527675","fields":{"phone":"01219928876","city":"Leeds"}}]}` |
-| `/field` [NOT IMPLEMENTED YET]                                            | `GET`    | Retrieves the field values of all API subscriptions                                            | `curl -v -X GET    "http://localhost:9650/field                                                                        -H "Cache-Control: no-cache"`                                                                                                                                         | 200 OK `{"subscriptions":[{"clientId":"hBnFo14C0y4SckYUbcoL2PbFA40a","apiContext":"ciao-api","apiVersion":"1.0","fieldsId":"f121ffa3-df94-43a0-8235-ac4530f9700a","fields":{"callback-url":"http://localhost:8080/callback","token":"abc59609za2q"}},{"clientId":"gGnFj14C05fScBYUbcoL3PbF540a","apiContext":"hello-api","apiVersion":"2.0","fieldsId":"ab1a03cf-8666-4e33-9ef7-07c172527675","fields":{"phone":"01219928876","city":"Leeds"}}]}`                 |
-| `/field/application/:client-id/context/:api-context/version/:api-version` | `DELETE` | Deletes the field values of an API subscription                                                | `curl -v -X DELETE "http://localhost:9650/field/application/hBnFo14C0y4SckYUbcoL2PbFA40a/context/ciao-api/version/1.0" -H "Cache-Control: no-cache"`                                                                                                                                         | 404 NOT_FOUND / 204 No Content                                                                                                                                                                                                                                                                                                                                                                                                                                    |
+## PUT Field Definitions 
+### `PUT /definition/context/:apiContext/version/:apiVersion`
+Creates or updates the definitions of the subscriptions fields for an API.
 
+### Response with
 
-## Tests
```

Comment:
You deleted the tests section.
Could you bring it back, on top the readme?

Reply:
done